action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   dispatch_event_type:
     description: "The name of the repository dispatch event, see https://git.io/JTpRo. Defaults to '[owner]/[repo] release', e.g. 'gr2m/release-notifier-action release'"
     required: false
+  embed_payload:
+    description: "Instead of event_payload, use a syntetic event object with payload embedded as JSON string, e.g. '{event_payload:<payload_as_json_string>}"
+    required: false
+    default: "false"
```

Comment:
I wouldn't set any value as default for optional boolean inputs, it could easily be interpreted as the string "false" which might be interpreted as truthy if not careful.

Reply:
Agree, will update PR and remove default value.